eng/build.ps1

```diff
@@ -259,7 +259,9 @@ function BuildSolution() {
   $roslynUseHardLinks = if ($ci) { "/p:ROSLYNUSEHARDLINKS=true" } else { "" }
 
   $restoreUseStaticGraphEvaluation = $true
-  
+
+  $symbolPublishingExcludeFile = Join-Path $RepoRoot "eng\SymbolPublishingExclusionsFile.txt"
```

Comment:
I'm not sure if this is needed, it might automaticallybe picked up (if this came from my original PR, it was probably just me trying to figure things out)

See https://github.com/dotnet/arcade/blob/main/Documentation/CorePackages/Publishing.md#can-we-exclude-symbols-from-publishing-to-symbols-server

Reply:
Reverted. New test run queued.